kolibri/core/templatetags/kolibri_tags.py

```diff
@@ -36,15 +39,51 @@ def kolibri_main_navigation():
 
 def render_as_tags(bundle):
     tags = []
-    static = getattr(settings, 'STATIC_URL')
     for chunk in bundle:
-        url = chunk.get('publicPath') or chunk['url']
         if chunk['name'].endswith('.js'):
-            tags.append('<script type="text/javascript" src="{static}/{url}"></script>'.format(static=static, url=url))
+            tags.append('<script type="text/javascript" src="{url}"></script>'.format(url=render_as_url(chunk)))
```

Comment:
When module assets are loaded synchronously here (or asynchronously via `scriptjs`), it looks like they are executed in the global namespace. However files like `example_module.js` seem to leverage `module.exports` for namespacing. Is `module.exports` actually being used?